src/main/java/com/aws/greengrass/clientdevices/auth/configuration/RuntimeConfiguration.java

```diff
@@ -47,6 +48,7 @@ public final class RuntimeConfiguration {
     static final String THINGS_KEY = "clientDeviceThings";
     static final String THINGS_V1_KEY = "v1";
     static final String THINGS_CERTIFICATES_KEY = "c";
+    static final String THINGS_VERSION_KEY = "v";
```

Comment:
what is the purpose of this?

We're already versioning by using the parent key.

Reply:
What is the purpose of that version?

Reply:
The idea is that this would be used for optimistic locking on Thing updates, though the original idea is that we wouldn't need to persist it. But, now I'm actually not seeing how it could be avoided.

We may want to rethink this one. I'd like to keep this memory footprint as small as we can.

Reply:
We'll get rid of this Thing version.